### ISSUE TITLE
Make sure you are using the correct path fon the en folder

### ISSUE THREAD

**Issue by Gomyde** [AntonioDePau/KHPCPatchManager#36]
When I try installing any mods, I get this message. It doesn't even give me an option to select a folder.
![image](https://user-images.githubusercontent.com/22133378/217113639-4803729f-b5d7-490f-88f3-aea8ef625ca2.png)

**Comment by liangste332** [comment]
I am getting the same message when I attempt to use the patch manager. I don't see a way or option to change or select a path to the en folder after I figured out where the en folder was located.

**Comment by liangste332** [comment]
I looked at the instructions for dragging and dropping patches and the en folder regarding the Birth by Sleep Ultimate Mix mod, but I am currently stuck as dragging and dropping the en folder onto the patcher does nothing.

**Comment by AntonioDePau** [comment]
Is the game installed on a drive other than C:/? Do you have an empty install folder for the game in the C:/ drive?

**Comment by liangste332** [comment]
No, my 1.5/2.5 game is installed in my C:/ drive.

**Comment by liangste332** [comment]
It's an odd situation for my modding experience so far. I dragged a joke translation patch for KH1.5 onto the patch manager and it appears to have worked given the edited KH1.5 opening logo screen, even though I couldn't get the 'drag en folder' process to work for me. Same for some KH2.5 patches that appear to have been successfully applied (voice lines for Lingering Will and Xemnas cut dialouge), interestingly enough.

It's the BBS patches that are giving me trouble because I'm not sure what to do about the en folder and the 'resources' folder full of text files that the patch manager keeps generating. I cannot get the BBS patches to go through the applied process.

On a side note, I was also struggling to figure out that lua backend hook installation, as I followed the instructions for the installation of the three critical lua files but they are not installed correctly, given that F2 and F1 aren't brining up any console for me.

**Comment by AntonioDePau** [comment]
If your game(s) is installed in the official location you shouldn't be running into any issues: **C:\Program Files\Epic Games\**KH_1.5_2.5\

Can you confirm the path of your KH 1.5 + 2.5 and KH 2.8 games?

**Comment by liangste332** [comment]
C:\Program Files\Epic Games\KH_1.5_2.5

**Comment by Xemnoraq** [comment]
yeah i get the same issue, for me it doesnt seem to recognize my steam install or something, it was working fine for epic games, i originally had my steam version installed on a different drive so i moved it back to my C drive in the default steam installation pathway and the patch manager even asked me to click the folder and no matter what i do it doesn't seem to work, has anybody else figured this out? thankyou!

**Comment by Xemnoraq** [comment]
@AntonioDePau Hey man, appreciate what you do for the modding community, i've been able to run some really fun mods with KH patch manager, for some reason though with the steam install it doesn't seem to recognize it even when i point out the folder when it asks me to, i'm using the latest release of the KH patch manager, in my above comment i mentioned that i originally had my KH1 installed on my E-drive, and i thought that may have been causing it because on the E-drive the directory is: SteamLibrary/SteamApps/Common/KINGDOM HEARTS -HD 1.5+2.5 ReMIX-

So i moved it to the default pathway on my C-drive as: C:\Program Files (x86)\Steam\steamapps\common

When i tried to apply i always get this message:

KHPCPatchManager v1.2.1
Applying KH1 patch...
If you want to patch KH1, KH2, Recom or BBS, please drag your "en" or "dt" folder (the one that contains kh1_first, kh1_second, etc.) located under "Kingdom Hearts HD 1 5 and 2 5 ReMIX/Image/" or "Steam/steamapps/common/KINGDOM HEARTS -HD 1.5+2.5 ReMIX-/Image/" here, and press Enter:
"C:\Program Files (x86)\Steam\steamapps\common\KINGDOM HEARTS -HD 1.5+2.5 ReMIX-\Image\dt"
Could not find any folder to patch!
Make sure you are using the correct path for the "en" or "dt" folder!


not sure why its doing this or what might be going wrong but any help would be awesome, thanks man :)

**Comment by AntonioDePau** [comment]
Hey @Xemnoraq,

I downloaded the latest version just now and tried with both GUI and CLI. It works on my end with the Steam version installed in the  documents folder of my D drive.

A few questions for you:

1. Are you using the GUI interface of KHPCPatchManager, or using it through CLI?

2. If using the GUI (if not, please try GUI by just double-clicking the exe without dragging a patch file onto it), when prompted to browse and select the relevant folder, are you selecting the "KINGDOM HEARTS -HD 1.5+2.5 ReMIX-" folder, as per the below screenshot?
![Image](https://github.com/user-attachments/assets/a5121abe-777c-4b15-8e45-406ea803c135)

3. If using the CLI (if not, please try CLI by dragging a patch file onto the exe), when prompted to drag or paste the path to the "dt" folder, are you entering the whole path up to and including the "dt" folder (for Steam) as below?
![Image](https://github.com/user-attachments/assets/8c05ade6-4570-4d2e-bd9a-5481ef2c81d4)

4. Go to your "dt" folder (where the .hed and .pkg files are located) and take a screenshot where the following are clearly visible (see below screenshot):
    - the .hed and .pkg files of the game you are attempting to patch
    - the path to the folder you are in

![Image](https://github.com/user-attachments/assets/ec5ab7ba-3b24-4881-97a4-e198d8657754)

You may want to try uninstalling the game(s) and re-installing from scratch (in the default path) to see if that works. Maybe it's a Windows issue where the path is incorrectly detected/registered?
